### PR TITLE
Add exporter resource entry in cluster helm chart

### PR DESCRIPTION
**Pull request by blackliner**
**Description of your changes:**

Add resource entry in helm chart for:
- exporter

relevant sourcecode:
- https://github.com/rook/rook/blob/0a7c3c17747c8962677a5678164207c38bff30c7/pkg/apis/ceph.rook.io/v1/resources.go#L47

**Which issue is resolved by this Pull Request:**
Resolves N/A

**Checklist:**

- [x] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure)).
- [ ] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [x] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [x] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

**Review by travisn — COMMENTED** [review]
@blackliner These requests and limits were based on what you see running?
@avanthakkar Do you have an idea about the memory/cpu the exporter would be expected to consume?

**Comment by galexrt** [comment]
@blackliner Could you also update the `cluster.yaml` and `cluster-test.yaml` example files? Please let us know if you are available to update the PR, so we can take over otherwise, thanks!

**Comment by galexrt** [comment]
@avanthakkar ping regarding @travisn comment.

@blackliner are you available to update the PR? If not, please let us know so we can tackle this change in a separate PR.

File: deploy/charts/rook-ceph-cluster/values.yaml [hunk 1]
```diff
@@ -331,6 +331,13 @@ cephClusterSpec:
       requests:
         cpu: "500m"
         memory: "100Mi"
+    exporter:
```
Comment (parth-gr) [inline deploy/charts/rook-ceph-cluster/values.yaml:334]: it is ceph exporter?

Comment (galexrt) [reply deploy/charts/rook-ceph-cluster/values.yaml:334]: @parth-gr Yes, https://github.com/rook/rook/blob/ae0823e52aa717ff816242ecd0b5ebd8c8405c83/pkg/operator/ceph/cluster/nodedaemon/exporter.go#L179

**Comment by galexrt** [comment]
I have opened #12551 to take over the changes in this PR.

**Pull request closed without merging**